### PR TITLE
Adjusted SAM Site missile launch offset

### LINKED ISSUE #13788: RA SAM Site missiles launch from beneath the turret
A LocalOffset should be added to the Armament so that the missiles correctly launch from the turret.

ra/rules/structures.yaml
SAM:
->Armament:
-->LocalOffset: 0,0,320

ra/sequences/structures.yaml
sam:
->muzzle: samfire
-->Offset: -1,6

### PR DESCRIPTION
**Pull request by abc013**
Closes #13788.

### CODE REVIEW

**Comment by ltem** [comment]
The weapon armament offset looks right but the muzzle offset seems to be a bit off. 
I mean to remember that the muzzle offset depends on the armament offset, so if you fix one the other one is off.

muzzleoffset

File: mods/ra/sequences/structures.yaml [hunk 1]
```diff
@@ -438,7 +438,7 @@ sam:
 	muzzle: samfire
 		Length: 18
 		Facings: 8
-		Offset: -1,-2
+		Offset: -1.6,-2
```
Comment (reaperrr) [inline mods/ra/sequences/structures.yaml]: This should be `-1, 6` (and I mean the whole thing, `-2` needs to be replaced with `6`).

Comment (abc013) [inline mods/ra/sequences/structures.yaml]: Updated.

**Review by SoScared — APPROVED** [review]
Missile launch height looking way more accurate compared to the current.

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
Simple and risk-free enough to go into stable, I think.

[ef41840](https://github.com/OpenRA/OpenRA/commit/ef418403c58cf809894ff70009e522f93ff85f37)

https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/c53380430bbc7991ee9f04f541448511f698a9aa...91ae6a892d8df00b5c54d7b24d76bae2e95fb304

**Pull request merged**